EpsilonAuxBMS/Src/ReportAuxStatusToCanTask.c

```diff
@@ -23,16 +23,14 @@ void reportAuxStatusToCanTask(void const* arg)
         // Set CAN message address
         hcan1.pTxMsg->StdId = AUX_STATUS_STDID;
         // Set Data
-        hcan1.pTxMsg->Data[0] = 0;
-        hcan1.pTxMsg->Data[0] += auxStatus.commonContactorState * 0x1;
-        hcan1.pTxMsg->Data[0] += auxStatus.chargeContactorState * 0x2;
-        hcan1.pTxMsg->Data[0] += auxStatus.dischargeContactorState * 0x4;
-        hcan1.pTxMsg->Data[0] |= auxStatus.auxVoltage * 0x8;
-        // Initialize to 0 so there's no garbage in [2] to [7]
-        hcan1.pTxMsg->Data[1] = 0;
-        hcan1.pTxMsg->Data[1] += auxStatus.strobeBmsLight * 0x1;
-        hcan1.pTxMsg->Data[1] += auxStatus.allowCharge * 0x2;
-        hcan1.pTxMsg->Data[1] += auxStatus.contactorError * 0x4;
+        hcan1.pTxMsg->Data[0] += auxStatus.commonContactorState;
```

Comment:
these should all be ORd together. The semantics of `+=` for bitwise operations is wrong.